src/client/src/actions/wizardInfoActions/updateDependencyInfo.ts

```diff
@@ -0,0 +1,20 @@
+import { WIZARD_INFO_TYPEKEYS } from "./typeKeys";
+
+export interface IUpdateDependencyInfo {
+  type: WIZARD_INFO_TYPEKEYS.UPDATE_DEPENDENCY_INFO;
+  payload: any;
+}
+
+export const updateDependencyInfoAction = (
+  dependencyInfo: any
```

Comment:
do you think it would be a good idea if we had some type safety for the dependencyInfo?

Possibly something like:
```ts
interface IDependencyInfo {
    dependency: "python" | "node";
    installed: boolean;
}
```